### DISCUSSION TITLE
view and change user groups

### DISCUSSION THREAD

**Discussion by amgschnappi** [elementary/settings-useraccounts#195] [Ideas]
Please increase the functionality in this panel.
Checking usergroups or adding user to usergroups is a big headache for GUI person like me.

**Comment by jeremypw** [comment]
I am transferring this to the `switchboard-plug-useraccounts` project issues, although I am not sure if exposing advanced functionality here is in line with design or security  policy.  As a work around you can install `gnome-system-tools` which will put several tools in the Applications/System menu including Uses and Groups.   That package does not seem to be exposed in the AppCenter but you can install with the Terminal command `sudo apt install gnome-system-tools`.  These tools are ugly but powerful. Take care not to damage your system.

**Comment by amgschnappi** [comment]
1. On desktop PC & laptops, windows is the most dominant OS. As desktop OS, its daily usage is more than monthly or even yearly usage of Linux desktop. Windows has successfully exposed this advanced functionality to many home and office users without any/huge negative impact.

Then why is eOS afraid of such things?

Your decision (to not expose advanced functionality - least people mess up their systems) seems to be imaginary ghost, which doesnt exist in reality.

2. Out of the box, eOS provides mail & web apps. Normal people want FF/Chrome & Thunderbird. Flatpak is not enabled by default. Is flatpak, with much more apps & much more support behind it, some advanced thing?

Your decisions are quite contradictory. On one end you dont want to provide "advanced" user mgmt (which already Microsoft has proved, is not a problem) to normal non techie users, and then you expect these normal non techie users to dive into cli & do some cli wizardry to get basic tools like FF& TB?

**Reply by danirabbit** [reply]
Please try to keep conversation scoped to a single issue. It makes it really hard for the dev team to track and address concerns if unrelated information is scattered around between different discussions. If you have a question about sideloading please open a separate discussion for that.

Regarding user groups can you provide more information about what it is you're trying to achieve by changing user groups? I think Jeremy is right that we want to be careful not to create a setting that you could use to hurt yourself, but if we had more information about the end goal then I think we can try to design a feature that is safe and efficient

**Comment by amgschnappi** [comment]
I want to add Guest user to few groups like cdrom via GUI, without playing the cli piano.

**Reply by danirabbit** [reply]
Let's focus on the end result that you'd like to achieve here instead of the steps you would take to get there. Is the issue here that the Guest account is missing access to certain features by default? That is, would the issue be solved if the Guest account was already part of those groups out of the box? Are there other accounts that you're experience permissions issues with or is this just an issue with the Guest account?

**Comment by amgschnappi** [comment]
I would also like to add my main user to virtual box group and docker group. Let's just say that having user group s funtionality as a GUI will make my life easier.

**Comment by amgschnappi** [comment]
I also want to have a separate account with sudo priviliges for my wife on this laptop. We would be having our personal folders and also shared folders. All this is so much easier via GUI.
